api/http/handler/user.go

```diff
@@ -397,8 +397,13 @@ func (handler *UserHandler) handleDeleteUser(w http.ResponseWriter, r *http.Requ
 		return
 	}
 
-	if userID == 1 {
-		httperror.WriteErrorResponse(w, portainer.ErrCannotRemoveAdmin, http.StatusForbidden, handler.Logger)
+	users, err := handler.UserService.UsersByRole(portainer.AdministratorRole)
```

Comment:
Thing is, we already prevent the removal of an administrator account where the ID of the user to remove equals the ID of the user executing the removal request (thus preventing an administrator to remove its account). I believe that this is just what we need, no need to check if at least 2 admin account exists before authorizing the removal.

You can just remove the `if userID == 1`  block.